acceptance/features/scaling.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Scaling down broker nodes
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Skipping incremental scale downs
+    Given I create a basic cluster "scaledown" with 5 nodes
+    And cluster "scaledown" is stable with 5 nodes
+    When I scale "scaledown" to 3 nodes
+    Then cluster "scaledown" should be stable with 3 nodes
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Scaling up nodes
```

Comment:
This test covers the same ground as `resource-redpanda-crud` which was essentially just setting some random fields and then scaling up the cluster.